src/components/variant/ui/drawer-window.tsx

```diff
@@ -0,0 +1,234 @@
+import { MouseEvent, ReactElement, useRef, useState } from 'react'
+import ScrollContainer from 'react-indiana-drag-scroll'
+import Checkbox from 'react-three-state-checkbox'
+import cn from 'classnames'
+import { get } from 'lodash'
+import { observer } from 'mobx-react-lite'
+import Tooltip from 'rc-tooltip'
+
+import { IGridLayout } from '@declarations'
+import { t } from '@i18n'
+import {
+  ICommonAspectDescriptor,
+  IPreAspectDescriptor,
+  ITableAspectDescriptor,
+  TRecCntResponse,
+} from '@service-providers/dataset-level/dataset-level.interface'
+
+const normClass = 'norm'
+const normHitClass = 'norm hit'
+const noTrHitClass = 'no-tr-hit'
+
+const PreView = ({
+  content,
+}: ICommonAspectDescriptor & IPreAspectDescriptor): ReactElement => {
+  return <pre className="overflow-y-hidden">{content}</pre>
+}
+
+interface ITableViewProps
+  extends ICommonAspectDescriptor,
+    ITableAspectDescriptor {
+  shouldAddShadow: boolean
+}
+
+const TableView = ({
+  colhead,
+  rows,
+  name,
+  shouldAddShadow,
+}: ITableViewProps): ReactElement => {
+  let colheadData: string[] = []
+
+  if (colhead) {
+    colheadData = [colhead?.[0]?.[0]]
```

Comment:
you can push element without reassigning it, so you can use `const colheadData`

Reply:
Same situation. I didnt change this logick in this PR